docs/user_guides/single_table/copulagan.rst

```diff
@@ -688,19 +688,23 @@ Conditional Sampling
 
 As the name implies, conditional sampling allows us to sample from a conditional
 distribution using the ``CopulaGAN`` model, which means we can generate only values that
-satisfy certain conditions. These conditional values can be passed to the ``conditions``
-parameter in the ``sample`` method either as a dataframe or a dictionary.
+satisfy certain conditions. These conditional values can be passed to the `sample_conditions`
```

Comment:
The previous user guide used double \`\` instead of single `, so I think we should use double `` for consistency. Maybe wait to see what @csala has to say about it, since I don't know which one we want to use either.

Reply:
That sounds good to me, I'll make the change.